Code/Tools/SceneAPI/SceneData/Behaviors/ScriptProcessorRuleBehavior.cpp

```diff
@@ -377,11 +415,34 @@ namespace AZ::SceneAPI::Behaviors
             }
         };
 
-        [[maybe_unused]] Events::ProcessingResult fallbackResult;
-        if (LoadPython(context.GetScene(), scriptPath, fallbackResult))
+        EditorPythonConsoleNotificationHandler logger;
+        m_editorPythonEventsInterface->ExecuteWithLock(executeCallback);
+    }
+
+    bool ScriptProcessorRuleBehavior::DoPrepareForExport(Events::PreExportEventContext& context)
+    {
+        using namespace AzToolsFramework;
+
+        if (LoadPython())
+        {
+            auto defaultScript = FindMatchingDefaultScript(context.GetScene());
+            if (defaultScript)
+            {
+                SignalScriptForExportEvent(context, defaultScript.value());
+            }
+
+            [[maybe_unused]] Events::ProcessingResult fallbackResult;
+            auto manifestScript = FindManifestScript(context.GetScene(), fallbackResult);
+            if (manifestScript)
+            {
+                SignalScriptForExportEvent(context, manifestScript.value());
+            }
+        }
+        else
         {
-            EditorPythonConsoleNotificationHandler logger;
-            m_editorPythonEventsInterface->ExecuteWithLock(executeCallback);
+            AZ_Warning("scene", false,
+                "The scene (%s) attempted to prepare Python but Python can not start",
```

Comment:
Is there any suggestion we can give to resolve this? I can't think of any, but figured I would check. If a content creator hits this, they might end up stuck not knowing what steps to take to resolve it.